ecr-public/get-credentials.sh

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env bash
```

Comment:
Want to add the license header to these new scripts?

Reply:
Done